features/account/customer_account/address_book/viewing_addresses.feature

```diff
@@ -0,0 +1,15 @@
+@customer_account
+Feature: Viewing a customer's addresses
+    In order to see all my addresses in address book
+    As a Customer
+    I want to be able to browse my addresses
+
+    Background:
+        Given the store operates on a single channel in "United States"
+        And I am a logged in customer
+        And I have an address "Lucifer Morningstar", "Seaside Fwy", "90802", "Los Angeles", "United States" in address book
+
+    @ui
+    Scenario: Viewing all addresses
```

Comment:
Can we also add another customer in the background with some addresses in the address book? In order to make sure that they see only their addresses. (that's +1 scenario at least)